libr/include/r_core.h

```diff
@@ -331,6 +331,10 @@ typedef struct r_core_t {
 	RList *ropchain;
 	bool use_tree_sitter_r2cmd;
 
+	/* maybe move this into RCore */
```

Comment:
i guess the comment is not relevant anymore :D

Reply:
i would place that change in a new PR

Reply:
we can just force merge it if that is the only change actually... we will have to wait anyway for the new PR for the comment :D and doing a PR just for a line of a comment seems a bit... meh!